TADashboard/src/test/java/tests/data_profiles/PositiveDPTests.java

```diff
@@ -0,0 +1,58 @@
+package tests.data_profiles;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.DashboardPage;
+import page_objects.DataProfilesPage;
+import page_objects.LoginPage;
+import tests.BaseTest;
+import utils.common.Constants;
+import utils.helper.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PositiveDPTests extends BaseTest {
+    List<String> preSetDataProfiles = new ArrayList<>();
+    LoginPage loginPage = new LoginPage();
+    DataProfilesPage dataProfilesPage = new DataProfilesPage();
+    DashboardPage dashboardPage = new DashboardPage();
+
+    @Test(description = "Verify that all Pre-set Data Profiles are populated correctly")
+    public void tc066_PreSetDataProfilesArePopulatedCorrectly() {
+        Logger.step("Login with a valid user");
+        loginPage.login(Constants.USER_NAME, "");
+
+        Logger.step("Go to Data Profiles page.");
+        dashboardPage.selectAdministerMenu("Data Profiles");
+
+        preSetDataProfiles.add("Action Implementation By Status");
+        preSetDataProfiles.add("Test Case Execution");
+        preSetDataProfiles.add("Test Case Execution Failure Trend");
+        preSetDataProfiles.add("Test Case Execution History");
+        preSetDataProfiles.add("Test Case Execution Results");
+        preSetDataProfiles.add("Test Case Execution Trend");
+        preSetDataProfiles.add("Test Module Execution");
+        preSetDataProfiles.add("Test Module Execution Failure Trend");
+        preSetDataProfiles.add("Test Module Execution History");
+        preSetDataProfiles.add("Test Module Execution Results");
+        preSetDataProfiles.add("Test Module Execution Results Report");
+        preSetDataProfiles.add("Test Module Execution Trend");
+        preSetDataProfiles.add("Test Module Implementation By Priority");
+        preSetDataProfiles.add("Test Module Implementation By Status");
+        preSetDataProfiles.add("Test Module Status per Assigned Users");
+        preSetDataProfiles.add("Test Objective Execution");
+
+        for (String dataProfileValue: preSetDataProfiles) {
```

Comment:
dataProfileValue -> dataProfile

Reply:
Yeah, I have been updated for it. Please check again